go.mod

```diff
@@ -24,7 +24,7 @@ require (
 	golang.org/x/crypto v0.0.0-20220722155217-630584e8d5aa
 	golang.org/x/sync v0.1.0
 	golang.org/x/sys v0.2.0
```

Comment:
This PR conflicts with the `main` branch since `golang.org/x/sys` is  now `v0.3.0`.
Can you rebase it?

Reply:
done

Reply:
Thanks!